### DISCUSSION TITLE
Commercial Use

### DISCUSSION THREAD

**Discussion by kyleskom** [kyleskom/NBA-Machine-Learning-Sports-Betting#234] [Announcements]
It has come to my attention that this repo is being used to start business and websites that sell the predictions for a profit. This repository has no license and without a license, the default copyright laws apply, meaning that I retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work. 
[Source](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository)

This project is currently free for all and using this project to make money is unaccetable. 

I will be DMCA copyrighting your websites. 

This will not apply to anyone who freely uses this for personal use or contributes to this project.

**Comment by onlyadamscott** [comment]
Not surprising. Glad you made this clear. This is a research tool essentially.

**Comment by nealmick** [comment]
this is quite concerning.

**Reply by kyleskom** [reply]
What part is concerning?

**Reply by nealmick** [reply]
I don't see a way of preventing this sort of thing from happening, my project is open source as-well.  If someone creates a basic website and just rebrands everything or just a basic blog applications posting predictions, I have no way of telling.  Even if the project is not open source, and let's say you pay-wall the project.  Anyone could easily then go create a scam telegram group or whatever, where people pay to get in and they just post your predictions :(

**Reply by sonymag** [reply]
agree=)

**Reply by kyleskom** [reply]
Ya fully preventing this 100% isn't feasible. Legal action is always an option though if one explicitly prevents commercial use.  My intention here is to not let people get ripped off from my work that is free for everyone to use here.

**Reply by onlyadamscott** [reply]
Bruh....stop it

**Reply by onlyadamscott** [reply]
I'm over here training models and you here with this noise?

**Reply by kyleskom** [reply]
@onlyadamscott what

**Reply by onlyadamscott** [reply]
not you sir.

**Reply by onlyadamscott** [reply]
Sorry. Watching people attack this git hurts my feelings, but expected.

**Comment by sonymag** [comment]
@kyleskom Thanks for listening to me, friend

**Comment by aaaclones** [comment]
@kyleskom this is the trade off that comes a long with making projects public and open source on one hand you acquire great ideas and were able to fix a lot of problems the downside that comes with the open source idea is the bad actors that do it purely for profit an gain , not sure if you can have it both ways ... going after each and everyone of them with copyrights action will just be very taxing and exhausting .. as long as you own the source codes that's great . At some certain point when your able to finally figure out the project in totality maybe then you can just make it private or consider it , but zero chance you can shut every website down that's gaining from your work

**Reply by kyleskom** [reply]
I 100% agree.

**Comment by a461204a** [comment]
I found a youtuber using your program as their own profit
https://youtu.be/oCGRlKyOGrY?t=370
This video pushes all program sources to GPT
But looking at the displayed results, it is obvious that it is your program.

**Reply by nealmick** [reply]
pretty scary,  guy has an instagram posting pick's too, linked below description.  I assume this guy is selling or profiting some how, as he docent link the project.  Although i did translate the page, it didn't seem like alot of people were talking about selling picks or memberships.... Also the instagram has something like a linktree posted, which has a few links, one of them for donations, another for telegram.  The program he is using in the video is this project no doubt.

**Reply by aaaclones** [reply]
Open source means open source this is bound to happen and aside from this , this project is nowhere near fully optimized .. so in the case the model does well they will garner somewhat of traffic but the  minute the model starts to hallucinate all the people will just abandon it .. ultimately it's Kyles decision, but he made it open source he can't expect just the good without any sort of blowback

**Comment by DillonMercz** [comment]
![Screenshot 2024-04-25 140453](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/16828120/63f1f4fe-6d0b-4880-85ad-2d4720151174)



This website is using your stuff https://betvision.ai

**Reply by kyleskom** [reply]
Does that website work for you?

**Reply by kyleskom** [reply]
Ah I see it now you provided wrong url

**Reply by kyleskom** [reply]
Whats the discord

**Reply by DillonMercz** [reply]
Sorry, url is https://betvisionai.com

**Reply by DillonMercz** [reply]
https://discord.gg/ptmkGShw

**Reply by kyleskom** [reply]
I cant find that post or anything that seem malicious

**Reply by DillonMercz** [reply]
It was in my DMs, I was asking him what he uses for NBA, and he sent the repo and said something about having a dude install this project on his server.

**Reply by DillonMercz** [reply]
Let's keep an eye on this
